README.md

```diff
@@ -1899,13 +1899,124 @@ and verify your domain. Generate an API key for the Mailgun account. Note your M
+```
+
+##### Required Mailgun keys:
+
+| Name           | Descriptor       | How you get it                                                   | 
+|----------------|------------------|------------------------------------------------------------------|
+| `key`          | `MAILGUN_KEY`    | passed in through .env files or secrets.                         |
+| `domain`       | `Mailgun Domain` | passed into application.yaml as a string into the domain field.  |
+| `sender-email` | `Sender Email`   | passed into the application.yaml file in the sender-email field. | 
+
+##### Required Fields for Emails
+
+To build an email message you must pass at least three String arguments into the sendEmail()
+method a MailgunEmailClient object:
+
+1. an email subject
+2. an email recipient
+3. an email body. The email body will take a string written in html and apply the text and styling
+   to the body of the email.
+
+* Note: We suggest passing the emails subject and the email body through message.properties to allow
+  for language translation.
+
+Each email message can also include these fields:
+
+1. emailsToCC → This is a list of emails passed into the cc: field of application.yaml.
+   To capture the list of cc's, developers can grab the list of emails from the application.yaml
+   using
+   @Value and passing in the field ”${form-flow.flow.ubi.email.confirmation.cc:}” based on the
+   application.yaml found above.
+2. email Bcc → is the list of emails passed into the bcc: field of application.yaml
+3. attachments → The list of files to attach, This field accepts a list of files that are added as
+   attachments to the email.
+4. requireTls → this is a boolean that determines whether a message can only be sent through tls
+   connection.  
+   True indicates that a message can only be using TLS encription.
+
+| Name            | Input Type   | Required Field                | Description                                                 | Defaults   | 
+|-----------------|--------------|-------------------------------|-------------------------------------------------------------|------------| 
+| subject         | String       | *                             | The subject of an email                                     |            |
+| recipient email | String       | *                             | The mailbox that the email is sent to                       |            | 
+| emailToCC       | List<String> | *                             | The list of mailboxes that are copied the email             | empty list |
+| emailToBcc      | List<String> |                               | List of mailboxes that are blind copied the email           | empty list | 
+| emailBody       | String       |                               | Body of an email.  The html version                         |            |  
+| attachments     | List<File>   | * Required to add attachments | List of files to be added as attachment to an email         | empty list | 
```

Comment:
Not sure it's necessary to call out attachments is required if you want to add attachments? Makes the wording a little confusing as attachments themselves are not required.

Reply:
done